sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -171,222 +186,263 @@ Sample code snippets are provided to illustrate using long-running operations [b
+        )
+    )
 
-poller = form_recognizer_client.begin_recognize_content(form)
-form_pages = poller.result()
+    for line_idx, line in enumerate(page.lines):
+        print(
+            "...Line # {} has content '{}' within bounding box '{}'".format(
+                line_idx,
+                line.content,
+                line.bounding_box,
+            )
+        )
 
-for content in form_pages:
-    for table in content.tables:
-        print("Table found on page {}:".format(table.page_number))
-        print("Table location {}:".format(table.bounding_box))
-        for cell in table.cells:
-            print("Cell text: {}".format(cell.text))
-            print("Location: {}".format(cell.bounding_box))
-            print("Confidence score: {}\n".format(cell.confidence))
+    for word in page.words:
+        print(
+            "...Word '{}' has a confidence of {}".format(
+                word.content, word.confidence
+            )
+        )
 
-    if content.selection_marks:
-        print("Selection marks found on page {}:".format(content.page_number))
-        for selection_mark in content.selection_marks:
-            print("Selection mark is '{}' within bounding box '{}' and has a confidence of {}".format(
+    for selection_mark in page.selection_marks:
+        print(
+            "...Selection mark is '{}' within bounding box '{}' and has a confidence of {}".format(
                 selection_mark.state,
                 selection_mark.bounding_box,
-                selection_mark.confidence
-            ))
+                selection_mark.confidence,
+            )
+        )
+
+for table_idx, table in enumerate(result.tables):
+    print(
+        "Table # {} has {} rows and {} columns".format(
+            table_idx, table.row_count, table.column_count
+        )
+    )
+    for region in table.bounding_regions:
+        print(
+            "Table # {} location on page: {} is {}".format(
+                table_idx,
+                region.page_number,
+                region.bounding_box
+            )
+        )
+    for cell in table.cells:
+        print(
+            "...Cell[{}][{}] has content '{}'".format(
+                cell.row_index,
+                cell.column_index,
+                cell.content,
+            )
+        )
 ```
 
 ### Using Prebuilt Models
-Extract fields from certain types of common forms such as receipts, invoices, business cards, and identity documents using prebuilt models provided by the Form Recognizer service.
+Extract fields from select document types such as receipts, invoices, business cards, and identity documents using prebuilt models provided by the Form Recognizer service.
 
-For example, to extract fields from a sales receipt, use the prebuilt receipt model provided by the `begin_recognize_receipts` method:
+For example, to extract fields from a sales receipt, use the prebuilt receipt model provided by passing `model="prebuilt-receipt"` into the `begin_analyze_documents` method:
```

Comment:
Again probably noticing these words a lot more now, should analyze be used here instead of extract?